src/task/models/tileMergeTaskManager.ts

```diff
@@ -104,25 +111,28 @@ export class TileMergeTaskManager {
 
       const logger = this.logger.child({ jobId, jobType, taskType: this.taskType });
       let taskBatch: ICreateTaskBody<MergeTaskParameters>[] = [];
+      let lastInsertedTaskIndex: MergeTaskParameters['taskIndex'];
```

Comment:
you probably mean: `IJobResumeState`;
`let lastInsertedTaskIndex: IJobResumeState`

Reply:
btw its not only the last inserted task index but zoom level too:
would rename it for: `jobResumeState`